### PR TITLE
Fixes staging and Fixes #593

### LINKED ISSUE #593: Add search input in top nav
@umairidris would love to get your thoughts on this too. @ahmedre suggest we bring it back. I'd image it to look like:

image

### PR DESCRIPTION
**Pull request by mmahalwy**
Fixes #593

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33198

File: src/locale/ur.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default {
     'search.placeholder': 'تلاش کر یں',
     'search.resultHeading': '{from}-{to} OF <span> {total} </span> SEARCH RESULTS FOR: <span>{query}</span>', // eslint-disable-line max-len
     'search.error': 'معذرت، آپ کی تلاش کے ساتھ ایک خرابی پیش آگئی.',
+    'search.nothing': 'کوئی تلاش کے سوال',
```
Comment (naveed-ahmad) [inline src/locale/ur.js]: a better translation for "no search query" in Urdu would be "تلاش کے لئے کچھ لکھئے "

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32773

**Comment by mmahalwy** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32774

**Pull request merged**